### PR TITLE
Add Chest entity

### PR DESCRIPTION
**Pull request by 2Steaks**

### CODE REVIEW

File: actors/player.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Player:
     def __init__(self, name):
         self.health = 100
-        self.name = name
\ No newline at end of file
+        self.name = name
+        self.inventory = []
```
Comment (chasinglogic) [inline actors/player.py]: This is a lint failure, you need a newline at the end of file in python.

**Review by chasinglogic — CHANGES_REQUESTED** [review]
If you use `black` on that file it will fix it.

**Review by sophiedaviespatrick — APPROVED** [review]
Looks good AFAICS

**Review by chasinglogic — APPROVED** [review]

File: entities/chest.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                return True
+
+        if action == "Take all":
+            actor.inventory.extend(self.inventory)
+            return False
+
+        if action == "Close":
+            self.is_open = False
+            return True
+
```
Comment (davidttest) [inline entities/chest.py:27]: If action is not Take all / Close, and assuming the inventory is not empty, we would want to extend the actor's inventory?

File: entities/chest.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        self.is_open = False
+
+    def do(self, action, actor):
+        if not self.is_open:
+            if action == "Yes":
+                self.is_open = True
+                return False
+            else:
+                return True
```
Comment (davidttest) [inline entities/chest.py:18]: Do we want to make sure the user entered Yes or No?

**Review by davidttest — APPROVED** [review]
A couple of comments that would be nice to have

**Review by mikehiett-mpb — APPROVED** [review]

**Pull request merged**